tests/features/support/step-definitions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Cucumber step definitions for Nightwatch
+ * @see {@link https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/step_definitions.md|Cucumber JS step definitions}
+ */
+
+const { defineStep, defineParameterType } = require('cucumber');
+const i = require('./step-runners');
+
+defineParameterType({
+  name: 'target',
+  regexp: /`[^`]+`/,
+  transformer: (value) => {
+    return value.slice(1, -1);
+  }
+});
+
+defineStep('I browse/open/visit (a/an/the)( ){target}', (pageName) =>
+  i.openAPage(pageName));
+defineStep('I find/identify/see/spot (a/an/the)( ){target}', (qa) =>
+  i.seeATarget(qa));
+defineStep('I find/identify/see/spot (a/an/the)( ){target} in/on the {target}', (qa, parentQa) =>
+  i.seeATarget([qa, parentQa]));
+defineStep('I find/identify/see/spot (a/an/the)( ){target} with the text {string}', (qa, text) =>
+  i.seeATargetWithText(qa, text));
+defineStep('I can\'t/don\'t find/identify/see/spot (a/an/the)( ){target}', (qa) =>
```

Comment:
we might want a negative step like this for the more specific selector. "I can't see the 'logo' in the 'search result'"

Reply:
```
defineStep('I can\'t/don\'t  find/identify/see/spot (a/an/the)( ){target} in/on the {target}', (qa, parentQa) =>
  i.doNotSeeATarget([qa, parentQa]));
```